pkg/vrraw/vrConsts_test.go

```diff
@@ -0,0 +1,181 @@
+package vrraw_test
+
+import (
+	"github.com/suyashkumar/dicom/pkg/vrraw"
+	"testing"
+)
+
+// Tests that the VR consts are what we expect
+func TestVRRawValues(t *testing.T) {
+	type TestCase struct {
+		VR       string
+		Expected string
+	}
+
+	testCases := []TestCase{
```

Comment:
I would recommend using anonymous structs for tests cases, as this is a common convention throughout Go code (in the stdlib too). It also reduces boilerplate code a bit.

See https://github.com/golang/go/wiki/TableDrivenTests#example-of-a-table-driven-test

Reply:
No problem, I think I tend to lean away from anonymous structs as I prefer the self-documenting / explicit nature of giving structs a name, but that's just a stylistic preference on my part and a little out of line with how a lot of Go is written. I'll switch out my table tests on other PRs to use anonymous structs as well.

Reply:
Thanks! Yeah, my feeling is that in this case, it's typically better to stick with idiomatic code patterns found throughout the Go ecosystem (e.g. in [stdlib](https://github.com/golang/go/wiki/TableDrivenTests#example-of-a-table-driven-test) and other common Go libraries)--this means the code tends to be more "familiar" at first glance to folks working in Go (and this pattern is quite common in my experience--and is common throughout this library).

As for the readability of unnamed structs, I think the thinking is that the struct is only relevant to that particular test scope, so it may not be that important what the name is (similar to the reasoning behind Go's short variable names for variables used in a small well defined scope: https://github.com/golang/go/wiki/CodeReviewComments#variable-names